.github/workflows/cd.yml

```diff
@@ -0,0 +1,33 @@
+name: Continuous Deployment
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          prerelease: ${{ contains(github.ref, '-') }}
```

Comment:
Could you elaborate on how this determines if something should be marked as a prerelease?

Reply:
See here: https://semver.org/#spec-item-9. For example `v1.0.0-alpha`.

Reply:
I see, LGTM